pyro/optim/dct_adam.py

```diff
@@ -33,6 +31,20 @@ def _transform_inverse(x, dim, duration):
     return idct(x[right], dim)[left].add_(x[left])
 
 
+def _get_mask(x, indices):
```

Comment:
i'm having a bit of trouble understanding this helper. can you please either add more comments or add a test?

Reply:
I just add some comments to explain the logic. :)